src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -72,7 +72,7 @@ internal static IExpression CreateEvaluatorDirect(CheckResult result, StackDepth
             result.ThrowOnErrors();
 
             (var irnode, var ruleScopeSymbol) = IRTranslator.Translate(result._binding);
-            return new ParsedExpression(irnode, ruleScopeSymbol, stackMarker);
+            return new ParsedExpression(irnode, ruleScopeSymbol, stackMarker, result.CultureInfo);

```

Comment:
>result.CultureInfo); [](http://example.com/codeflow?start=78&length=20)

@anderson-joyle - we're seeing cases where this is null. 
Carlos will reach out.